test/common/http/http1/BUILD

```diff
@@ -60,7 +60,8 @@ envoy_cc_test(
         "//test/mocks/http:http_mocks",
         "//test/mocks/network:network_mocks",
         "//test/mocks/runtime:runtime_mocks",
-        "//test/mocks/upstream:upstream_mocks",
+        "//test/mocks/upstream:cluster_info_mocks",
+        "//test/mocks/upstream:transport_socket_match_mocks",
```

Comment:
was this a transitive or missing dep?

Reply:
the `transport_socket_match_mocks` target was a missed dep and I added it into rules